### PR TITLE
Feat/registration/financial aid

### PR DESCRIPTION
**Pull request by josix**
## Types of Changes

**Thanks for sending a pull request! Please fill in the following content to let us know better about this change.**
Please put an `x` in the box that applies.

- [ ] **Bugfix**
- [x] **New feature**
- [ ] **Refactoring**
- [ ] **Breaking change** (any change that would cause existing functionality to not work as expected)
- [ ] **Documentation Update**
- [ ] **Other (please describe)**

## Description
- [x] Add H1, Hr, TwoColumnWrapper Component 
- [x] Add ApplicationHowTo Diagrams
- [x] Complete Registration/Financial Aid Page (Web Ver.)
- [x] RWD version

## Steps to Test This Pull Request

1. Go to 'http://localhost:3000/pycontw-2021/en-us/registration/financial-aid'

## Expected Behavior

localhost_3000_pycontw-2021_zh-hant_registration_financial-aid (1)

localhost_3000_pycontw-2021_zh-hant_registration_financial-aid

## Related Issue

If applicable, reference to the issue related to this pull request.

## More Information

**Screenshots**

**Additional context**
Add any other context about the problem here. You may also want to refer
to [how to write the perfect pull request](https://github.blog/2015-01-21-how-to-write-the-perfect-pull-request/).

### CODE REVIEW

**Comment by mattwang44** [comment]
[Add FA page](https://trello.com/c/Afuf1zNI/39-add-fa-page)

File: pages/registration/financial-aid.vue [hunk 1]
```diff
@@ -1,11 +1,191 @@
...
+                </p>
+                <p class="content paragraph">
+                    {{ $t('coreValueContent')[1] }}
+                </p>
+            </template>
+        </two-col-wrapper>
+        <core-hr></core-hr>
+        <two-col-wrapper
+            v-for="(field, i) in $t('plainTextFields')"
+            :key="`field_${i}`"
```
Comment (mattwang44) [inline pages/registration/financial-aid.vue:27]: `field_<index>` is quite common and could be duplicated in other pages. Please follow our current naming convention: `<page name>_<component/section name>_<index>`.

File: pages/registration/financial-aid.vue [hunk 2]
```diff
@@ -1,11 +1,191 @@
...
+                <ul class="list-disc">
+                    <li class="content">
+                        {{ $t('fill') }}
+                        <ext-link
+                            href="https://forms.gle/vXZNwRPhu84G4NM49"
+                            highlight
+                            underline
+                        >
+                            {{ $t('form') }}
+                        </ext-link>
```
Comment (mattwang44) [inline pages/registration/financial-aid.vue:81]: The following change can remove the excessive space.

```suggestion
                        <ext-link
                            href="https://forms.gle/vXZNwRPhu84G4NM49"
                            highlight
                            underline
                        >{{
                            $t('form')
                        }}</ext-link>
```

File: pages/registration/financial-aid.vue [hunk 3]
```diff
@@ -1,11 +1,191 @@
...
+        </two-col-wrapper>
+        <two-col-wrapper>
+            <template #default>
+                <p>{{ $t('remarkTitle') }}</p>
+            </template>
+            <template #right-col>
+                <ul class="list-disc">
+                    <li class="content">
+                        {{ $t('fill') }}
+                        <ext-link
```
Comment (mattwang44) [inline pages/registration/financial-aid.vue:75]: The link can be inserted with [slot syntax](https://kazupon.github.io/vue-i18n/guide/interpolation.html#slots-syntax-usage)

File: nuxt.config.js [hunk 4]
```diff
@@ -10,7 +10,9 @@ export default {
     router: {
         base: process.env.ROUTER_BASE || DEFAULT_ROUTER_BASE,
     },
-
+    server: {
```
Comment (mattwang44) [inline nuxt.config.js:13]: why do we need these lines?

Comment (josix) [reply nuxt.config.js:13]: Oops I pushed the workaround script for mobile phone layout checking. I will fix it next commit also update to use tailwind css styling.

**Comment by josix** [comment]
@mattwang44 Thanks for your advice. I've corrected the problems mentioned above. Please have a look, thank you!

**Review by mattwang44 — APPROVED** [review]
LGTM

**Pull request merged**